tools/csv-merger/csv-merger.go

```diff
@@ -1004,8 +1002,9 @@ func getUXBackendServerDeployment() appsv1.DeploymentSpec {
 							"-tls-key=/etc/tls/private/tls.key",
 							"-cookie-secret-file=/etc/proxy/secrets/session_secret",
 							"-openshift-service-account=ux-backend-server",
-							`-openshift-delegate-urls={"/":{"resource":"StorageCluster","namespace":"openshift-storage","verb":"create"}}`,
-							"-openshift-ca=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"},
+							`-openshift-delegate-urls={"/":{"group":"ocs.openshift.io","resource":"storageclusters","namespace":"openshift-storage","verb":"create"}}`,
+							"-openshift-ca=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt",
+						},
```

Comment:
Is this somehow related to the problem you're solving? If so please add some reasoning to your commit message.

Reply:
There is new BZ raised for this particular issue now and handled in separate PR added the description for it as well. https://github.com/red-hat-storage/ocs-operator/pull/2397